src/hpc/traverse.h

```diff
@@ -8,12 +8,35 @@
  */
 
 typedef void (*TraversalFunction)(Obj);
+typedef void (*TraversalCopyFunction)(Obj copy, Obj original);
+
+typedef enum {
+    TRAVERSE_NONE,
+    TRAVERSE_BY_FUNCTION,
+    TRAVERSE_ALL,
+    TRAVERSE_ALL_BUT_FIRST,
+} TraversalMethodEnum;
+
+//
+// TODO
+//
```

Comment:
What is it that needs to be done?

Reply:
Ah, I wanted to add a comment there explaining what the function does. I'll remove this here, and add that comment in a future PR (I already have a bunch of them which I wrote while trying to understand the traverse code, and fixing some of it)